### PR TITLE
Move to simpler brute force substring search

### PR DESCRIPTION
**Pull request by jacquarg**
Please review #136 before this one !

Remove full text but plain words pouchdb-quick-search plugin and replace it with a systematic substring search in each folder and files names.

Speed seems decent : a few second on Motog (gen1), with 1400 files and folders.

### CODE REVIEW

**Comment by poupotte** [comment]
@kosssi, @aenario, Can you review this PR (after #136) , please ?

File: www-src/app/collections/files.coffee [hunk 1]
```diff
@@ -18,21 +18,20 @@ module.exports = class FileAndFolderCollection extends Backbone.Collection
...
-    # search use temporary view
+
     search: (callback) ->
-        params =
-            query: @query
-            fields: ['name']
-            include_docs: true
+        app.replicator.db.query 'FilesAndFolder', (err, all) =>
+            results = all.rows.filter (row) =>
+                return row.key[1].indexOf(@query) isnt -1
```
Comment (kosssi) [inline www-src/app/collections/files.coffee]: tu n'as pas besoin du return

File: www-src/app/views/menu.coffee [hunk 2]
```diff
@@ -50,6 +50,7 @@ module.exports = class Menu extends BaseView
     doSearchIfEnter: (event) => @doSearch() if event.which is 13
     doSearch: ->
         val = $('#search-input').val()
+        val = val.toLowerCase()
```
Comment (kosssi) [inline www-src/app/views/menu.coffee]: val = $('#search-input').val().toLowerCase() ?

File: www-src/app/collections/files.coffee (hunk 1 above)
Comment (jacquarg) [inline www-src/app/collections/files.coffee]: true. 
The advice of [Cozy-guideline](https://github.com/cozy/cozy-guidelines) about this is fun : ` the return keyword should be explicit when needed `

Comment (kosssi) [inline www-src/app/collections/files.coffee]: ok

**Pull request merged**